src/m365/util/commands/accesstoken/accesstoken-get.ts

```diff
@@ -50,23 +48,25 @@ class UtilAccessTokenGetCommand extends Command {
     );
   }
 
-  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     let resource: string = args.options.resource;
-    if (resource.toLowerCase() === 'sharepoint') {
-      if (auth.service.spoUrl) {
-        resource = auth.service.spoUrl;
-      }
-      else {
-        return cb(`SharePoint URL undefined. Use the 'm365 spo set --url https://contoso.sharepoint.com' command to set the URL`);
+
+    try {
+      if (resource.toLowerCase() === 'sharepoint') {
```

Comment:
This clause should go outside of the `try..catch` so that the exception won't get processed by `handleRejectedODataJsonPromise`

Reply:
Indeed, I'll adjust it 😃